src/main/java/frc/robot/subsystems/auto/BargeAlign.java

```diff
@@ -0,0 +1,113 @@
+package frc.robot.subsystems.auto;
+
+import com.ctre.phoenix6.swerve.SwerveModule.DriveRequestType;
+import com.ctre.phoenix6.swerve.SwerveRequest;
+import com.ctre.phoenix6.swerve.SwerveRequest.ForwardPerspectiveValue;
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.drivebase.CommandSwerveDrivetrain;
+
+public class BargeAlign extends Command {
+  private PIDController pidX = new PIDController(4, 0, 0);
+  private PIDController pidRotate = new PIDController(8, 0, 0);
+
+  private CommandSwerveDrivetrain drive;
+  private static final double fieldLength = 17.548; // Welded field
+  private static final double blueBlacklineX = 7.09;
+  private static final double redBlacklineX = fieldLength - blueBlacklineX;
+  private static final double blueBargeLineX = 7.67;
+  private static final double redBargeLineX = fieldLength - blueBargeLineX;
+
+  private final SwerveRequest.FieldCentric blackLineDriveRequest =
+      new SwerveRequest.FieldCentric() // Add a 10% deadband
```

Comment:
? What does the `Add a 10% deadband` comment refer to?

Reply:
Copy-pasta from the controls drive request, which in turn is from the original generated CTRE source. We should definitely remove those stale comments.